### PR TITLE
[JAVA]SpringThrowArnab.java

### PR DESCRIPTION
**Pull request by codebook-2000**
The code for [Spring Throw](https://github.com/dscnsec/DSC-NSEC-Algorithms/commit/920778137a6ac6f9729cc2f86d5282a353493e5e) accomplishes the following points:-

- [X] Code is added in right directory(Folder).
- [X] Brief Description of Solution is given.
- [X] Time and Space Complexity is mentioned
- [X] Code is well indented.
- [X] Comments are clearly written wherever necessary.
- [X] Code is successfully compiled and passes all test cases.
- [X] Variables naming is verbose.

### CODE REVIEW

File: 1. Array/push_zero/PushZeroArnab.java.orig [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/**
+ * PushZero.java
```
Comment (OverLordAct) [inline 1. Array/push_zero/PushZeroArnab.java.orig]: What is this file?

File: 7. Dynamic Programming/spring_throw/SpringThrowArnab.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+public class SpringThrowArnab {
+    static int solve(int[] arr, int n) {
+        //ct stores the count of no of throws
+        //max stores the maximum distance upto which every spring can send the person
+        //current is the point uptill where person can go by the string after which it needs to get updated
+
+        int j = 0, ct = 0, max = 0, current = 0;
+        while (j < n - 1) {  //calculating till n-2 as after it it will directly be at last index
+            max = Math.max(max, arr[j] + j);//storing the maximum of all distances it can travel
```
Comment (OverLordAct) [inline 7. Dynamic Programming/spring_throw/SpringThrowArnab.java:24]: In your solution, I believe you are not checking if the block as reached the nth block or not, but rather if it is reaching nth block and beyond. Can you check the problem statement as it requires you to reach the nth block and not beyond.

File: 1. Array/push_zero/PushZeroArnab.java.orig (hunk 1 above)
Comment (codebook-2000) [reply 1. Array/push_zero/PushZeroArnab.java.orig]: > What is this file?

Actually there was a merge conflict coming, after that whwn I resolved that conflict using mergetool, this fuile of push zero is getting committed every time, And I don't know how to remove it

**Comment by codebook-2000** [comment]
Can you check once if this commit [Updating some changes](https://github.com/dscnsec/DSC-NSEC-Algorithms/pull/71/commits/8b755108aae9df2d6021d1268bc9825e6d487a1c) meets your expectations?

**Comment by codebook-2000** [comment]
I have deleted the file, Can you please check once?

**Pull request merged**